frontend/src/app/modules/workspace/personal/personal.component.ts

```diff
@@ -53,34 +55,27 @@ export class PersonalComponent implements OnInit {
     .pipe(takeUntil(this.unsubscribe$))
     .subscribe((user) => (this.currentUser = user));;
  }
+ createFolder(name: string) {
+   let newfolder : NewFolderDto = {
+   name: name,
+   parentId: this.currentFolder,
+   authorId: this.user,
+   teamId: this.team
+ }
 
-  private validateForm() {
-    this.folderForm = this.formBuilder.group({
-      name: [, {
-        validators: [
-          Validators.required
-        ],
-        updateOn: 'change',
-      }],
-    });
-  }
-
-  createFolder() {
-    let newfolder : NewFolderDto = {
-      name: this.folderForm.controls['name'].value,
-      parentId: this.currentFolder,
-      authorId: this.user,
-      teamId: this.team
-    }
-
-    this.folderService.add(newfolder).subscribe((response) => {
+   this.folderService.add(newfolder).subscribe((response) => {
       this.folder = response.body as FolderDto;
-      this.isFolderFormShow = false;
-      this.folderForm.setValue({ 'name': ''});
     });
   }
 
   showNewFolderForm() {
-    this.isFolderFormShow = true;
+    const dialogRef = this.dialog.open(FolderDialogComponent, {
+      width: '250px',
+      data: {name: 'Untitled folder'},
```

Comment:
'Untitled folder' must be a placeholder, not a hardcoded string. With this approach user has to remove it before writing his own name for the folder